assets/sass/config/_variables.scss

```diff
@@ -16,64 +16,64 @@
  * limitations under the License.
  */
 
+@import "variables-mui3";
+
 /*
  * Colors
  * Names from http://chir.ag/projects/name-that-color/
```

Comment:
@techanvil That is correct, color names are out of sync. What we did is just tweaking the existing colors to use the colors from the new color palette in case which are GM3 compatible. These colors are no longer meant to be used now, nor new color variables should be added since we should exclusively use colors in `assets/sass/config/_variables-mui3.scss` going forward.